nuitka/plugins/standard/DataFileCollectorPlugin.py

```diff
@@ -35,6 +35,7 @@
     "scrapy": ((None, "VERSION"),),
     "requests": (("certifi", "../certifi/cacert.pem"),),
     "importlib_resources": ((None, "version.txt"),),
+    "coincurve": ((None, "./libsecp256k1.dll"),),
```

Comment:
When a module needs a DLL, that should only be the case, when it is not linked against it, otherwise Nuitka will normally detect static DLL usage.

If this is a case of a dynamically opened DLL you ought to add it to the short handling in `considerExtraDlls` in the same file.

Reply:
Thank you, I will take a look at this "considerExtraDlls". Sadly, it is not bundled automatically.

Reply:
I tested this, but I have no idea what is the correct format, any hints?

```
        #Support for coincurve
        if full_name == "coincurve":
            coincurve_dll_path = locateDLL("libsecp256k1.dll")
            dist_dll_path = os.path.join(dist_dir, os.path.basename(coincurve_dll_path))
            return ((coincurve_dll_path, dist_dll_path, None),)

        return ()
```

Reply:
I missed this unfortunately. It looks wrong to provide the ".dll" suffix, that will make it non-portable, also it seems you might want to check for the `coincurve._libsecp256k1._dll` module, for enhanced clarity. Otherwise it looks good to me. Can you rebase the PR? if that's too painful, I would do it, I probably caused the troubles with my rebases of develop. Doing `git rebase upstream/develop` and skipping conflicting commits except your own ones will work though.